### PR TITLE
just-random

### PR DESCRIPTION
**Pull request by EvandroLG**
It produces a random number between two values.

### CODE REVIEW

File: test/function-random/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  t.ok(expected(random(), 0, 1));
+  t.ok(expected(random(), 0, 1));
+
+  t.end();
+});
+
+test('returns a number between 5 and 10', function(t) {
+  t.plan(5);
+
+  t.ok(expected(random(5, 5, 10), 5, 10));
```
Comment (angus-c) [inline test/function-random/index.js:23]: I think you added extra arg by mistake!

File: packages/function-random/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "just-random",
+  "version": "1.0.0",
+  "description": "Produces a number number between two values",
+  "main": "index.js",
```
Comment (angus-c) [inline packages/function-random/package.json:5]: Please add `"types": "index.d.ts",` under this line

File: test/function-random/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  t.ok(expected(random(5, 5, 10), 5, 10));
+  t.ok(expected(random(5, 5, 10), 5, 10));
+  t.ok(expected(random(5, 5, 10), 5, 10));
+  t.ok(expected(random(5, 5, 10), 5, 10));
+
+  t.end();
+});
+
+test('returns a number from 0 to 5', function(t) {
```
Comment (angus-c) [inline test/function-random/index.js:32]: Can you add one more test with negative arguments

**Review by angus-c — CHANGES_REQUESTED** [review]
Thanks! We're going to have to rename this `just-random-number` because there is already a `just-random` for arrays.

Also I think it belongs in numbers section not function? I see why you did function section, but all current function utils take a function as an argument, and all current number utils take numbers

**Comment by Masa-Shin** [comment]
IMHO the name `number-random`  should be reserved for a package that helps to get random number (float) in a range. How about `number-random-int` or `number-random-integer` for the package?

**Comment by angus-c** [comment]
@Masa-Shin I agree, `just-random-int` is good for package name and `number-random-int` for internal name

**Review by angus-c — APPROVED** [review]

**Comment by angus-c** [comment]
Thank you!
